go.mod

```diff
@@ -291,25 +291,18 @@ require (
 	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/apiextensions-apiserver v0.23.4 // indirect
-	k8s.io/gengo v0.0.0-20220307231824-4627b89bbf1b // indirect
-	k8s.io/klog v1.0.0 // indirect
-	k8s.io/klog/v2 v2.60.1 // indirect
-	k8s.io/legacy-cloud-providers v0.24.1 // indirect
+	k8s.io/apiextensions-apiserver v0.23.8 // indirect
+	k8s.io/gengo v0.0.0-20220613173612-397b4ae3bce7 // indirect
+	k8s.io/klog/v2 v2.70.1 // indirect
+	k8s.io/legacy-cloud-providers v0.23.9 // indirect
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9 // indirect
+	sigs.k8s.io/json v0.0.0-20211208200746-9f7c6b3444d2 // indirect
 	sigs.k8s.io/release-utils v0.6.0 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
 replace (
-	github.com/containerd/containerd => github.com/containerd/containerd v1.5.10
 	github.com/prometheus/common => github.com/prometheus/common v0.26.0
-	k8s.io/api => k8s.io/api v0.20.11
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.20.11
-	k8s.io/apimachinery => k8s.io/apimachinery v0.20.11
-	k8s.io/client-go => k8s.io/client-go v0.20.11
-	k8s.io/code-generator => k8s.io/code-generator v0.20.11
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20210305001622-591a79e4bda7
```

Comment:
Should this be bumped as well to to at least the 1.22 branch? https://github.com/kubernetes/kube-openapi/tree/release-1.22

Reply:
release-1.22 downgrades the other k8s.io deps. updated it to match https://github.com/tektoncd/pipeline